spyder/api/utils.py

```diff
@@ -0,0 +1,1096 @@
+# -*- coding: utf-8 -*-
```

Comment:
this utils functions aren't specific for codefolding, I'll name it `utils/editor.py`